SimPEG/electromagnetics/static/spontaneous_potential/_utils.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+import discretize
+from discretize.utils import active_from_xyz
+from scipy.interpolate import NearestNDInterpolator, LinearNDInterpolator
+
+
+def readSeepageModel(fname, mesh=None, xsurf=None, ysurf=None):
```

Comment:
I'm not sure this has a place in SimPEG right now...

Reply:
no need to bring this in.